app/views/admin/splits/_feature_completions.html.erb

```diff
@@ -0,0 +1,40 @@
+<article class="InfoCard sc-m-v--l">
+  <div class="InfoCard-titleContainer">
+    <h4>Feature Completions</h4>
+  </div>
+  <hr class="InfoCard-divider">
+  <% if split.feature_gate? %>
+    <% if split.feature_completions.any? %>
+      <div class="InfoCard-description">
+        <table class="fs-VariantsTable DescriptionTable">
+          <thead>
+            <th>App</th>
+            <th>Version</th>
+          </thead>
+          <tbody>
+            <% split.feature_completions.each do |feature_completion| %>
+              <tr>
+                <td><%= feature_completion.app.name %></td>
+                <td><%= feature_completion.version %>+</td>
+              </tr>
+            <% end %>
+          </tbody>
+        </table>
+      <% else %>
+        <div class="EmptyTable EmptyTable--centered sc-p--l">
+          <p>
+            <strong>This feature has not been marked as complete for any app versions yet.</strong>
+            <br>
+            As a result, this feature is not visible to any clients that rely on feature completions (barring alpha tester overrides).
```

Comment:
i don't know the full rules of feature completions... is there something specific to `alphas` or non-final releases or do you just mean overrides in general?

Reply:
Specifically overrides that are marked with the alpha checkbox in the admin UI

Reply:
Although I think that web extension overrides also set that checkbox

Reply:
Yup it does.